__tests__/routes/applications.test.js

```diff
@@ -219,4 +219,103 @@ describe('Application Router', () => {
       });
     });
   });
+
+  // This block describes the expected behaviour of the POST request for submitting new mentor/mentee profile applications
+  describe('[POST] /application/new/:role', () => {
+    // create some mock variables to submit with each test
+    // split this to test endpoints separately (ie, valid and invalid mentor - valid and invalid mentee)
```

Comment:
@ashurik522 @matt-arofin Can you please remove these comments? Might be appropriate to add some notes in the readme, however?